src/meta/meta_track.c

```diff
@@ -458,3 +467,44 @@ __wt_meta_track_handle_lock(WT_SESSION_IMPL *session, bool created)
 	trk->created = created;
 	return (0);
 }
+
+/*
+ * __wt_meta_track_init --
+ *	Intialize metadata tracking.
+ */
+int
+__wt_meta_track_init(WT_SESSION_IMPL *session)
+{
+	WT_CONNECTION_IMPL *conn;
+
+	conn = S2C(session);
+	if (!FLD_ISSET(conn->log_flags, WT_CONN_LOG_ENABLED))
+		WT_RET(__wt_open_internal_session(conn,
+		    "metadata-ckpt", false, WT_SESSION_NO_DATA_HANDLES,
+		    &conn->meta_ckpt_session));
```

Comment:
Could you make it explicit that the session needs to be open at read committed isolation?